code/controllers/subsystem/lighting.dm

```diff
@@ -12,7 +12,7 @@ var/list/lighting_update_overlays  = list() // List of lighting overlays queued
 /datum/subsystem/lighting
 	name = "Lighting"
 	wait = 1
-	init_order = 1
+	init_order = -20
```

Comment:
There may be a reason for it to be sub-ceded after. For example, the mapping subsystem used to be INFINITY